src/ol/source/imagesource.js

```diff
@@ -47,11 +46,6 @@ ol.source.Image = function(options) {
    */
   this.resolutions_ = goog.isDef(options.resolutions) ?
       options.resolutions : null;
-  goog.asserts.assert(goog.isNull(this.resolutions_) ||
-      goog.array.isSorted(this.resolutions_,
-          function(a, b) {
-            return b - a;
-          }, true));
```

Comment:
Ah, I understand, my apologies. I suspect the way I've altered the rest of the code was causing this to fail in unintended ways then.